internal/backends/postgresql/postgresql.go

```diff
@@ -41,17 +41,26 @@ type stats struct {
 // collectionsStats returns statistics about tables and indexes for the given collections.
 //
 // If the list of collections is empty, then stats filled with zero values is returned.
-func collectionsStats(ctx context.Context, p *pgxpool.Pool, dbName string, list []*metadata.Collection) (*stats, error) {
-	if len(list) == 0 {
-		return new(stats), nil
-	}
-
+func collectionsStats(ctx context.Context, p *pgxpool.Pool, dbName string, list []*metadata.Collection, val bool) (*stats, error) {
 	var err error
+	if len(list) == 0 && val {
```

Comment:
The previous logic was correct, I think. `collectionStats` is always called with a list of collections. If the list is empty, then there are no collections in that database.

Reply:
I will make necessary changes.